### DISCUSSION TITLE
Adding column name to its respective value?

### DISCUSSION THREAD

**Discussion by SMOK1N-ACES** [h2oai/datatable#3187] [Q&A] [answered]
Is there a way for me to add the column name to the value in dataTable without a time consuming for loop? IE I want to add the column names to all of the values... IE "Physician_Profile_First_Name/Marlon", "Physician_Profile_Middle_Name/G"...
any thoughts? It can be hacky as long as I have that datatable speed. Big Thanks!

![SO](https://user-images.githubusercontent.com/30246600/138320458-f1daaf50-380d-4ee2-8f5f-b5d0d73f8d35.PNG)

**Comment by lmullany** [comment, marked as answer]
Hmm, is this the type of loop you were trying to avoid?
```
data = dt.Frame(First=["t","r","s"],Middle=["p","q","r"],Last=["a","b","c"])
for col in data.names:
    data[col] = data[:, col + " / " + f[col]]
```

output:
```
   | First      Middle      Last    
   | str32      str32       str32   
-- + ---------  ----------  --------
 0 | First / t  Middle / p  Last / a
 1 | First / r  Middle / q  Last / b
 2 | First / s  Middle / r  Last / c
```

**Reply by SMOK1N-ACES** [reply]
haha yes and no. I was trying to avoid the for loop because I thought it would make things so slow it would not work for my application. But after running your method I can see that is not the case.

Big thanks for the help.

**Reply by oleksiyskononenko** [reply]
Yeah, that's the loop over the columns only, and not over all the data. So it won't cause any significant performance drop.
